source/connect/in-use-encryption.txt

```diff
@@ -0,0 +1,233 @@
+.. _in-use-encryption-tab:
+
+=================
+In-Use Encryption 
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   In-Use Encryption is an Enterprise/Atlas only feature. 
+
+The :guilabel:`In-Use Encryption` connection tab allows you to connect your 
+deployments using :manual:`Queryable Encryption
+</core/queryable-encryption/>`. 
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`In-Use Encryption` tab.
+
+      .. procedure::
+         :style: connected
+
+         .. step:: Provide a :guilabel:`Key Vault Namespace`.
+
+            A :guilabel:`Key Vault Namespace` refers to a collection that 
+            contains all the data keys used for encryption and decryption. 
+
+            Specify a collection in which data encryption keys are stored in 
+            the format ``<db>.<collection>``.
+
```

Comment:
Might be worth a mention here, that the non-offical default database/collection for keyVault we used in the server documentation is `encryption.__keyVault`

Reply:
Done.